lib/dizzy.js

```diff
@@ -32,7 +32,7 @@ module.exports = (BulkProvider, DizzyProvider, util) => {
             var argsResolved;
 
             if (!Array.isArray(argsArray)) {
-                if (argsArray !== undefined && argsArray !== null) {
+                if (typeof argsArray === "object" && argsArray) {
```

Comment:
Line 34 ensures that argsArray is not an array.  If it is not the optional array, it could be the optional context, so this is trying to detect if it is a valid object.  If it is a valid object, the next few lines erase `argsArray` and set the context.